.circleci/config.yml

```diff
@@ -473,6 +487,15 @@ workflows:
             - prepare-python-3.6
             - build-synapse
 
+      - test:
+          name: test-fuzz-3.6
```

Comment:
Both this and `test-fuzz-3.7` needs to be referenced in the `requires` section of the `finalize` step in the workflow definitions.

Reply:
Done, I added the fuzz tests to the nightly build too.